src/main/scala/uk/gov/hmrc/playcrosscompilation/AbstractPlayCrossCompilation.scala

```diff
@@ -63,7 +68,8 @@ abstract class AbstractPlayCrossCompilation(
     },
     unmanagedResourceDirectories in Test += {
       (sourceDirectory in Test).value / playDir / "resources"
-    }
+    },
+    crossScalaVersions ~= playCorssScalaBuilds
```

Comment:
have you recompiled this? There is still a typo with Corss here: 
```
crossScalaVersions ~= playCorssScalaBuilds
```

Reply:
Rush mistake :/